libraries/AC_Avoidance/AP_OAPathPlanner.cpp

```diff
@@ -24,11 +24,11 @@
 extern const AP_HAL::HAL &hal;
 
 // parameter defaults
-const float OA_MARGIN_MAX_DEFAULT = 5;
-const int16_t OA_OPTIONS_DEFAULT = 1;
+static constexpr float OA_MARGIN_MAX_DEFAULT = 5;
+static constexpr int16_t OA_OPTIONS_DEFAULT = 1;
 
-const int16_t OA_UPDATE_MS = 1000;      // path planning updates run at 1hz
-const int16_t OA_TIMEOUT_MS = 3000;     // results over 3 seconds old are ignored
```

Comment:
We should be careful about making this timeout too short because on real hardware it does take time to calculate the path when using Dijkstra's.

Reply:
en. I think you are right! but shorting the planning period would be nice for local planning espically when driving in high-speed or dynamic enviroments

Reply:
If we change this timeout then we need to test how long it takes on our slowest board that supports OA (an F4 or F7?) with a large number of fence points and while the vehicle is flying (or at least simulated flying) so that the CPU is significantly stressed.

TBH, I wouldn't change this unless you're sure that it provides a real advantage.